tests/jballerina-unit-test/src/test/resources/test-src/types/never/never-type-negative.bal

```diff
@@ -0,0 +1,41 @@
+function testNeverTypedVariable() {
```

Comment:
Shall we add tests for the following scenarios as well?

- In a function with a `never` return type,
  - explicitly return `()`
  - explicitly return some other value (say, a `string`)

Reply:
Ack